src/clj/game/cards/assets.clj

```diff
@@ -580,10 +580,26 @@
                  :effect (effect (rfg-and-shuffle-rd-effect card 3))}]}
 
    "Jeeves Model Bioroids"
-   {:implementation "Trigger is manual"
-    :abilities [{:label "Gain [Click]"
-                 :msg "gain [Click]" :once :per-turn
-                 :effect (effect (gain :click 1))}]}
+    (let [jeeves (effect (gain :click 1))
+          ability {:label "Gain [Click]"
+                   :msg "gain [Click]"
+                   :once :per-turn
+                   :effect jeeves}
+          matrix  (atom {})]
```

Comment:
Thanks... need to read more on this - I expected everything inside the (lets) to be local.  Might have been fun on the live server!